bin/node-sass

```diff
@@ -229,63 +229,41 @@ function getOptions(args, options) {
  */
 
 function watch(options, emitter) {
-  var buildGraph = function(options) {
-    var graph;
-    var graphOptions = {
-      loadPaths: options.includePath,
-      extensions: ['scss', 'sass', 'css']
-    };
-
-    if (options.directory) {
-      graph = grapher.parseDir(options.directory, graphOptions);
-    } else {
-      graph = grapher.parseFile(options.src, graphOptions);
-    }
+  var handler = function(files) {
+    files.added.forEach(function(file) {
+      var watch = gaze.watched();
```

Comment:
Abstract was probably the wrong word, but here is a crude example example
```js
    var watch = watcher.watched()
      if (watch.indexOf(file) === -1) {
        watcher.add(file);
      }
    });
```
So the `watcher` module would export certain functions required in the bin